redwood-lazylayout-view/src/main/kotlin/app/cash/redwood/lazylayout/view/ViewLazyList.kt

```diff
@@ -101,9 +104,11 @@ internal open class ViewLazyList private constructor(
       addOnScrollListener(
         object : RecyclerView.OnScrollListener() {
           override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
+            val viewsOnScreen = recyclerView.children.map { recyclerView.getChildViewHolder(it).bindingAdapterPosition }
+            userHasScrolled = true // Prevent guest code from hijacking the scrollbar.
```

Comment:
Does this mean it's impossible to programmatically change the scroll position once the user has interacted with the list a single time?

Reply:
that's a good point ... yes, it is at the moment